Woof/Woof/View/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+/// A view displaying detailed information about the sitter.
+struct DetailPetSitterView: View {
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: DetailSitterViewModel
+
+    var body: some View {
+        VStack {
+            AvatarView(url: viewModel.imageURL)
+            FiveStarRatingView(rating: viewModel.rating)
+            TextWithIconLabelView(
+                iconName: Image.IconName.filledPerson,
+                text: viewModel.fullName
+            )
+            TextWithIconLabelView(
+                iconName: Image.IconName.phone,
+                text: viewModel.phoneNumber
+            )
+            Text(viewModel.bio)
```

Comment:
nit:

The bio has weird paragraphs.
Can we improve the look and feel here?
<img width="476" alt="Screenshot 2023-05-23 at 16 23 42" src="https://github.com/ios-course/ironfoudation-team-project/assets/6073510/91ec629b-ab58-4e0a-b1be-b390d05d883a">

Reply:
We have a multiline string for bio-section and this behaviour expects.
 bio: """
            I am a responsible and caring pet sitter who has been working with animals for several years.
            I have a deep understanding of their needs and behaviors and is committed to providing them with the
            best care possible. Emily enjoys spending time outdoors and taking pets for walks, and she always ensures
            that they get plenty of exercise and attention. With her friendly personality and reliable work ethic,
            Emily is a great choice for pet owners who want to ensure that their furry companions are in good hands.
            """

Reply:
So please make the issue to address it and share the issue in the conversation.

Reply:
The issue to fix this unreadable displaying of text:
https://github.com/ios-course/ironfoudation-team-project/issues/63